src/backend/tools/py.CLI/caliopen_cli/commands/import_vcard.py

```diff
@@ -0,0 +1,170 @@
+import os
+import vobject
+
+def import_vcard(username, directory, file_vcard, **kwargs):
+
+    from caliopen_main.user.parameters.contact import (NewContact, NewEmail,
+                                            NewPostalAddress, NewPublicKey, NewPhone,
+                                            NewOrganization, NewIM, Contact, PHONE_TYPES,
+                                            ADDRESS_TYPES, EMAIL_TYPES, KEY_CHOICES, ORG_TYPES,
+                                            IM_TYPES)
+
+    from schematics.types import UUIDType
+
+    from caliopen_main.user.parameters.types import PhoneNumberType
+    from caliopen_main.user.parameters.types import InternetAddressType
+
+    from caliopen_main.user.core.contact import Contact as CoreContact
+    from caliopen_main.user.core.user import User as CoreUser
+
+    new_contact = NewContact()
+
+    vcards = []
+
+    if directory:
+        files = [f for f in os.listdir(directory) if
+                     os.path.isfile(os.path.join(directory, f))]
+        for f in files:
+            ext = f.split('.')[-1]
+            if ext == 'vcard' or ext == 'vcf':
+                vcard = vobject.readOne(open(directory+'/'+f,'r'))
```

Comment:
no use of + for string concatenation
The preferred form is : '{directory}/{file>'.format(directory, f)